extensions/arc/src/typings/arc.d.ts

```diff
@@ -33,10 +33,12 @@ declare module 'arc' {
 		username: string,
 		rememberPassword: boolean,
 		resources: ResourceInfo[]
+
 	};
 
 	export interface IExtension {
-		getRegisteredDataControllers(): Promise<ControllerInfo[]>;
+		getRegisteredDataControllers(): Promise<{ label: string, info: ControllerInfo }[]>;
```

Comment:
I'd export this type to some alias since you use it elsewhere too

Reply:
I wanted by no suitable name came to my mind. I will think more :)